helm/cluster-vsphere/templates/ipam/_ipam.tpl

```diff
@@ -1,13 +1,13 @@
 {{/* vim: set filetype=mustache: */}}
 
 {{- define "isIpamControlPlaneIPEnabled" -}}
-    {{- if and (and (not .Values.global.connectivity.network.controlPlaneEndpoint.host) (.Values.global.connectivity.network.controlPlaneEndpoint.ipPoolName)) (.Capabilities.APIVersions.Has "ipam.cluster.x-k8s.io/v1alpha1/IPAddressClaim") }}
```

Comment:
Are you sure? The api version is still v1alpha1 in the cluster-api-app
https://github.com/giantswarm/cluster-api-app/blob/main/helm/cluster-api/files/core/bases/ipaddresses.ipam.cluster.x-k8s.io.yaml#L46

Reply:
There was a bug and we fixed it here https://github.com/giantswarm/cluster-api-app/pull/217

Reply:
Log from bootstrap cluster 
```
❯ helm history cluster-api -n giantswarm
WARNING: Kubernetes configuration file is group-readable. This is insecure. Location: /tmp/kind-mc-initial-gmc.kubeconfig
WARNING: Kubernetes configuration file is world-readable. This is insecure. Location: /tmp/kind-mc-initial-gmc.kubeconfig
REVISION	UPDATED                 	STATUS  	CHART             	APP VERSION	DESCRIPTION
1       	Tue Jul 30 07:42:03 2024	deployed	cluster-api-1.20.0	1.20.0     	Install complete
```

The fix is included in 1.20.1

Reply:
Nevertheless, this PR should be OK too.

Reply:
@erkanerol thank you for explaining where the CRD comes from - it confused me that it wasn't deployed by the ipam app. The error that I was seeing was indeed about the conversion webhook so the fix makes sense. This PR isn't really necessary to solve the issue then, but as you say it should be ok to merge anyway.